docs/contributing.rst

```diff
@@ -249,7 +281,8 @@ The contents of this file are *reStructuredText* formatted text that
 will be used as the content of the news file entry. You do not need to
 reference the issue or PR numbers here as *towncrier* will automatically
 add a reference to all of the affected issues when rendering the news
-file.
+file. Please also see ``CHANGES/README.rst`` for the correct format of
```

Comment:
This can be a link: https://github.com/aio-libs/aiohttp/tree/master/CHANGES#readme.

Reply:
I also like including such documents so they become a part of the Sphinx tree and their sections are linkable. Here's examples of what I did in other places:
* https://yarl.aio-libs.org/en/latest/contributing/guidelines/#adding-change-notes-with-your-prs
* https://ansible-pylibssh.rtfd.io/en/latest/contributing/guidelines/#adding-change-notes-with-your-prs
* https://setuptools.pypa.io/en/latest/development/developer-guide.html#making-a-pull-request

Reply:
Shall I just replace this whole section with the included file then?